config.toml

```diff
@@ -217,6 +217,7 @@ url = "https://v1-19.docs.kubernetes.io"
 [params.ui]
 # Enable to show the side bar menu in its compact state.
 sidebar_menu_compact = false
+sidebar_menu_foldable = true
```

Comment:
Can we add a comment as well what it does like the other keys

Reply:
Have added